packages/store/test/test-rankOrder.js

```diff
@@ -120,23 +120,6 @@ const sortedSample = harden([
   // Lexicographic arrays. Shorter beats longer.
   // Lexicographic records by reverse sorted property name, then by values
   // in that order.
-
```

Comment:
We need some multi code unit chars in these tests to make sure we're unicode safe.

Reply:
PR please with such tests? I'll happily review.

Reply:
I was thinking something along the lines of [Mathias' Pile of Poo Test](https://mathiasbynens.be/notes/javascript-unicode#poo-test): `Iñtërnâtiônàlizætiøn☃💩`